include/internal/endian.h

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
This should be a testutil header. It has no use in our libraries (even though I know that there's code that use the same technique)

Reply:
Done.

Reply:
I would disagree, since there is code in the library (siv128.c) that uses this technique. Why not consolidate?